recipes/jsonrpcclient/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jsonrpcclient" %}
+{% set version = "3.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c50860409b73af9f94b648439caae3b4af80d5ac937f2a8ac7783de3d1050ba9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - jsonrpc=jsonrpcclient.__main__:main
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - apply_defaults <1
+    - click <7
+    - jsonschema <4
```

Comment:
I think you just need ``pip`` and ``python`` here in host. Could you please try that?

Reply:
Pushed the change, checks are in progress.

Reply:
Checks passed.